sw/device/silicon_creator/lib/cert/asn1.h

```diff
@@ -0,0 +1,200 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+
+#ifndef __ASN1__
```

Comment:
wrong format for header, see here for correct example: https://cs.opensource.google/opentitan/opentitan/+/master:sw/device/silicon_creator/lib/boot_log.h;drc=62e67c9ab7d7e16a0987d15c7c63026752a23733;l=5